src/NuGet.Core/NuGet.Versioning/NuGetVersion.cs

```diff
@@ -184,5 +184,13 @@ public int Revision
         {
             get { return _version.Revision; }
         }
+
+        /// <summary>
+        /// Returns true if version is a SemVer 2.0.0 version
+        /// </summary>
+        public bool IsSemVer2
+        {
+            get { return this.ReleaseLabels.Count() > 1 || this.HasMetadata; }
```

Comment:
I'm not a SemVer buff -- so having a single release label is acceptable for SemVer 1.0.0, but more than one is not okay?